src/sage/combinat/words/word_generators.py

```diff
@@ -1276,10 +1276,9 @@ def StandardEpisturmianWord(self, directive_word):
         """
         if not isinstance(directive_word, Word_class):
             raise TypeError("directive_word is not a word, so it cannot be used to build an episturmian word")
-        epistandard = directive_word.parent()(\
-                self._StandardEpisturmianWord_LetterIterator(directive_word),
-                datatype='iter')
-        return epistandard
+        return directive_word.parent()(
+            self._StandardEpisturmianWord_LetterIterator(directive_word),
+            datatype='iter')
```

Comment:
I understand the point. But the variable seems to serve as a comment too: What is computed is `epistandard`.